packages/connect-web-bench/README.md

```diff
@@ -10,5 +10,5 @@ it like a web server would usually do.
 
 | code generator | bundle size        | minified               | compressed           |
 |----------------|-------------------:|-----------------------:|---------------------:|
-| connect        | 113,448 b | 49,872 b | 13,465 b |
+| connect        | 113,741 b | 50,030 b | 13,533 b |
```

Comment:
The number are only slightly bumped because of https://github.com/connectrpc/connect-es/pull/842/commits/b57115ce70005b256530815db2917fb6e4604984. 
So tree-shaking is working as expected via the "module" export condition - at least in esbuild.

Reply:
I can confirm that tree shaking also still works in webpack. Currently looking at parcel, though having problems getting parcel tree shaking at all.

Reply:
Further digging into parcel looks like the it would resolve to cjs in the old version and seems to resolve to esm with the new version. However, both don't seem to be doing any tree shaking either way.